pkgs/tools/misc/alt-server/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, lib, pkgs, autoPatchelfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "AltServer-Linux";
+  version = "0.0.5";
+
+  src = fetchurl {
+    url = "https://github.com/NyaMisty/AltServer-Linux/releases/download/v${version}/AltServer-x86_64";
+    sha256 = "sha256-C+fDrcaewRd6FQMrO443xdDk/vtHycQ5zWLCOLPqF/s=";
```

Comment:
I think you should use `hash` in place of `sha256`.

This is optional since it could be automatized treewide at some point.

The idea is to avoid using `sha256` as attribute and use `hash` everywhere.

Since there could be different hashing algorithms in use, this make sense.